Mail/Outgoing.pm

```diff
@@ -400,7 +400,10 @@ sub _rewrite_from {
         return 0;
     }
     my($cfg) = $self->internal_get_config;
-    if ($cfg->{allow_resend_from_re}) {
+    if ($cfg->{in_btest}) {
+        return 1;
+    }
+    elsif ($cfg->{allow_resend_from_re}) {
```

Comment:
We tend not to put `elsif` here, just `if`.

Reply:
Noted. How do you feel about postfix style (i.e., `return 1 if $cfg->{in_btest}`)?

Reply:
We have used it quite a bit. I like the line break:

```pl
return 1
    if bla;
```

I think it is readable enough.

Reply:
ok, that's what I remembered, but see a fair number of places that either don't use postfix or do with it on the same line. Wanted to confirm the current preferences :)

Reply:
I think you can shape our preferences. We have been doing more of:

```perl
if (bla) {
   one line;
}
```

We feel like the braces are important. We do that in JS.